config/locales/views/groupings/en.yml

```diff
@@ -0,0 +1,8 @@
+en:
+  views:
+    groupings:
+      manage: "Manage"
+      set_team_settings: "Set of teams settings"
+      drag_and_drop_student_avatars: "You can <b>drag and drop</b> the student avatars to move them from one team to another."
```

Comment:
@nwoodthorpe yeah I'm not a fan of bringing in styles either. Not sure what the best way approach this is?

Do we need to emphasize **drag and drop**